oag/src/main/java/org/owasp/oag/config/FileConfigLoader.java

```diff
@@ -69,4 +160,11 @@ protected MainConfig load(InputStream defaultSettingsStream, InputStream userCon
 
         return finalConfig;
     }
+
+    /**
+     * Enables support to load configuration file via unsafe https connections.
+     */
+    public void enableUnsafeHttps() {
```

Comment:
Ah crap, this should read "allowUnsafeHttp"!!!! ANd yes, then it is the right name...please correct the flag's name!